src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -165,6 +165,7 @@ export function EmbeddedPositionSearchPanel({
         sRegion= undefined,
         toolbarHelpId= undefined,
         showHelpLines=true,
+        selectionHelpText= undefined,
         coordinateSys : csysStr = 'EQ_J2000',
         sx:hipsTargetViewSx={},
     }= slotProps.hipsTargetView ?? {};
```

Comment:
This is something for later, I also ran into it in my last ticket to add showHelpLines: EmbeddedPositionSearchPanel should directly expose the props of HiPSTargetView in slotProps instead of defining them repeatedly here. We can then do something like `<HiPSTargetView {...defaults, ...slotProps?.hipsTargetView, ...overrides}/>` (where `overrides` are the stuff that EmbeddedPositionPanel must need for it to work)